jsk_panda_robot/jsk_panda_teleop/euslisp/move_to_initial.l

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env roseus
```

Comment:
> please add README.md to jsk_panda_teleop,
how to start start_panda_teleop_follower_{side/master}_side.launch ?
should we start dual_panda.launch? or not ?

README is integrated in top directory (`jsk_panda_robot`):
https://github.com/pazeshun/jsk_robot/blob/bea7e509a1a027169ea06f4df74fe908e8a41443/jsk_panda_robot/README.md#via-roseus
https://github.com/pazeshun/jsk_robot/blob/bea7e509a1a027169ea06f4df74fe908e8a41443/jsk_panda_robot/README.md#teleop
@ykawamura96 mainly wrote this README and I'm not sure, but maybe this is `jsk_denso_robot` style:
https://github.com/jsk-ros-pkg/jsk_robot/blob/b40df2f1086bfd8b0c76a83284a7a3a5ad9b85f6/jsk_denso_robot/README.md#start-ros-node
If you think this style is wrong, OK, I'll separate README.

> If we do not have to start dual_panda.launch, please include them from start_panda_teleop_xxx.launch

Currently, we cannot use `dual_panda.launch` as it is in `start_panda_teleop_follower_side.launch` because they use different controllers:
https://github.com/pazeshun/jsk_robot/blob/bea7e509a1a027169ea06f4df74fe908e8a4144parallelly3/jsk_panda_robot/jsk_panda_startup/launch/dual_panda.launch#L31
https://github.com/pazeshun/jsk_robot/blob/bea7e509a1a027169ea06f4df74fe908e8a41443/jsk_panda_robot/jsk_panda_teleop/launch/start_panda_teleop_follower_side.launch#L24
The former (`dual_panda_effort_joint_trajectory_controller`) accepts JointTrajectoryAction, the latter (`dual_arm_cartesian_pose_controller`) accepts endcoords of both arms.
Perhaps they can coexist, I'll try that.

> if not please explain them at README.md explicitly.

I think current README is OK, because using roseus and using teleop are written parallelly.

> when we start move_to_reset_pose.launch?

I want to ask @ykawamura96 too...
This script moves the robot to `reset-manip-pose`, but current teleop using Touch USB cannot be started from that pose, because endcoords orientation is very different from initial Touch USB state...

> if start_audio_capture can be used in cobotta robot, please put them in jsk_robot_common.

I thought this should be done in another PR making cobotta robot use sound.
Because current egg drilling system using cobotta and sound has many local diffs, I have to check which script should be placed which repository...
Please, please wait......

Reply:
> README is integrated in top directory (jsk_panda_robot):

oh, i missed it. that is ok.

> Currently, we cannot use dual_panda.launch as it is in start_panda_teleop_follower_side.launch because they use different controllers:

 I see, if we still wan to use teleop, then we will add `arg` to switch controller in `jsk_panda_startup`. and put teleop-specific program such as touch - usb in teleop package. If not, we keep teleop specific startup file in teleop repository.

 > if they can coexist, I'll try that.

I am afraid this is difficult...

Reply:
>>    when we start move_to_reset_pose.launch?
>
>I want to ask @ykawamura96 too...
>This script moves the robot to reset-manip-pose, but current teleop using Touch USB cannot be started from that pose, because endcoords orientation is very different from initial Touch USB state...

I asked @ykawamura96 and he said he did not use move_to_reset_pose.launch.
So I removed this.

>>    Currently, we cannot use dual_panda.launch as it is in start_panda_teleop_follower_side.launch because they use different controllers:
>
>I see, if we still wan to use teleop, then we will add arg to switch controller in jsk_panda_startup. and put teleop-specific program such as touch - usb in teleop package. If not, we keep teleop specific startup file in teleop repository.

Fixed